src/main/java/FriendAdder.java

```diff
@@ -0,0 +1,42 @@
+import java.util.ArrayList;
+
+public class FriendAdder {
+    /** Adds users to friend list by sending invitations, then accepting invitations.
+     *
+     * @param user1 the username of user1 who sends friend request
+     * @param user2 the username of user2 who receives/ accepts friend request
+     *
+     */
+
+    public void sendFriendRequest(Person user1, Person user2){
+        //user 1 sends friend request to user 2
+
+        ArrayList<Person> oR = user1.getOutgoingRequests();
+        oR.add(user2);
```

Comment:
i think these variable names (oR and iR) are a little confusing -- perhaps they could be renamed outgoing and incoming?